spec/acceptance/class_spec.rb

```diff
@@ -126,6 +126,8 @@ class { 'puppetboard::apache::conf':
         ldap_bind_dn => 'cn=user,dc=puppet,dc=example,dc=com',
         ldap_bind_password => 'password',
         ldap_url     => 'ldap://puppet.example.com',
+        ldap_require_group => true,
```

Comment:
Hi - Yes, I think this is pretty close. But we should test both cases, so one context for ldap auth, and test that the config file does not contain "Require ldap-group".

Then make another conext that sets those parameters explicitly, and test for the expected results.